frb_rust/src/thread.rs

```diff
@@ -1,4 +1,32 @@
-pub const WORKERS_COUNT: usize = 4;
+#[cfg(not(wasm))]
+fn get_thread_count() -> usize {
+    #[allow(unused_mut, unused_assignments)]
+    let mut thread_count: usize = 4;
+    #[cfg(feature = "thread-count-single")]
+    {
+        worker_count = 1;
+    }
+    #[cfg(feature = "thread-count-max")]
+    {
+        thread_count = std::thread::available_parallelism().unwrap().get();
+    }
+    thread_count
+}
+
+#[cfg(wasm)]
+fn get_worker_count() -> usize {
+    #[allow(unused_mut, unused_assignments)]
+    let mut worker_count: usize = 4;
+    #[cfg(feature = "worker-count-single")]
+    {
+        worker_count = 1;
+    }
+    #[cfg(feature = "worker-count-max")]
+    {
+        worker_count = 16;
```

Comment:
btw, why 16 instead of something like "number of cores"? (I guess because there is no such API in the web?)

However, I guess few users have 16 cpu cores on their computers. Thus is it possible we find an API similar to std::thread::available_parallelism

Reply:
I just found out that workerpool relies on web workers, which actually ARE true multithreading. I should change the code. Also, web workers make use of logical cores, using the same `std::thread::available_parallelism` would be valid on WASM worker pool I think.

Reference about web workers: https://stackoverflow.com/questions/11871452/can-web-workers-utilize-100-of-a-multi-core-cpu